.gitignore

```diff
@@ -1,6 +1,6 @@
 # Created by https://www.gitignore.io/api/flask
 # Edit at https://www.gitignore.io/?templates=flask
-
+config.py
```

Comment:
You're right to gitignore this if it contains your secrets, but without it another developer won't be able to run the app. I'd suggest adding the `python-dotenv` package and storing your secrets in a `.env` file as we did originally - you can then avoid checking in the `.env` file, but can add the names of the expected keys into the `.env.template` file so that future developers know which variables they'll need to change